CHANGELOG.md

```diff
@@ -7,8 +7,6 @@ release.
 
 ## Unreleased
 
-- Updated AWS Java Lambda guidance - using system properties.
-
 ### Semantic Conventions
```

Comment:
Does a subsection titled "Semantic Conventions" still make sense for a repository "semantic-conventions"? IMHO, no.

Reply:
#124